StanfordDependencies/CoNLL.py

```diff
@@ -39,7 +39,9 @@ class Token(namedtuple('Token', FIELD_NAMES_PLUS)):
     There are other fields but they typically won't be populated by
     StanfordDependencies. Fields are immutable.
 
-    See http://ilk.uvt.nl/conll/#dataformat for a complete description."""
+    See CoNLL-X shared task on Multilingual Dependency Parsing by Buchholz and Marsi(2006)
```

Comment:
Micro-nit: can you wrap this text so it passes flake8? (pretty sure the failing nosetests are unrelated)

https://travis-ci.org/dmcc/PyStanfordDependencies/jobs/506774535#L692

Reply:
Committed the changes.